x-pack/plugins/apm/public/components/app/service_inventory/service_list/ServiceListMetric.tsx

```diff
@@ -14,18 +15,24 @@ export function ServiceListMetric({
   series,
   valueLabel,
   comparisonSeries,
+  hideSeries = false,
 }: {
   color: 'euiColorVis1' | 'euiColorVis0' | 'euiColorVis7';
   series?: Coordinate[];
   comparisonSeries?: Coordinate[];
   valueLabel: React.ReactNode;
+  hideSeries?: boolean;
 }) {
-  return (
-    <SparkPlot
-      valueLabel={valueLabel}
-      series={series}
-      color={color}
-      comparisonSeries={comparisonSeries}
-    />
-  );
+  if (!hideSeries) {
+    return (
+      <SparkPlot
+        valueLabel={valueLabel}
+        series={series}
+        color={color}
+        comparisonSeries={comparisonSeries}
+      />
+    );
+  }
+
+  return <EuiText>{valueLabel}</EuiText>;
```

Comment:
```suggestion
  return <EuiFlexGroup gutterSize="none" alignItems="flexEnd">
      <EuiFlexItem>
        <EuiText size="s" textAlign="right">
          {valueLabel}
        </EuiText>
      </EuiFlexItem>
    </EuiFlexGroup>;
```

Adding flex group around the value in order to align the text right inside. Also made the text size smaller to make it consistent with the regular table value style.

**Before**

<img width="686" alt="CleanShot 2021-08-05 at 10 09 48@2x" src="https://user-images.githubusercontent.com/4104278/128317263-2ab489e1-0d44-44da-bf12-a4c4f30c6a80.png">

*After*

<img width="690" alt="CleanShot 2021-08-05 at 10 22 50@2x" src="https://user-images.githubusercontent.com/4104278/128317374-e561fcc5-4510-42a1-96d3-7536d466d6e1.png">